qemu/tests/cfg/slof_device_tree.cfg

```diff
@@ -0,0 +1,8 @@
+- slof_device_tree: install setup image_copy unattended_install.cdrom
```

Comment:
Seems the case is only test for linux guest. Add "only Linux" to cfg file is better.